public/video-ui/src/components/Icon.js

```diff
@@ -85,7 +85,7 @@ export default class Icon extends React.Component {
       
     return (
       <span className={props.className}>
-        <i className="icon responsive--primary">{props.icon}</i>
+        <i className="icon responsive--primary" onClick={props.onClick}>{props.icon}</i>
```

Comment:
Is this going to cause errors if we do not pass in onClick?

Reply:
I don't think so, `onClick` will just be `undefined`